pkgs/development/python-modules/python-heatclient/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, buildPythonApplication, fetchPypi
+, pbr, Babel, cliff, iso8601, osc-lib, prettytable, oslo-i18n, oslo-serialization, oslo-utils, keystoneauth1, python-swiftclient, pyyaml, requests, six
+, stestr, testscenarios, requests-mock
+}:
+
+buildPythonApplication rec {
+  pname = "python-heatclient";
+  version = "2.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b53529eb73f08c384181a580efaa42293cc35e0e1ecc4b0bc14a5c7b202019bb";
+  };
+
+  nativeBuildInputs = [
+    pbr
+  ];
+
+  propagatedBuildInputs = [
+    Babel
+    cliff
+    iso8601
+    osc-lib
+    prettytable
+    oslo-i18n
+    oslo-serialization
+    oslo-utils
+    keystoneauth1
+    python-swiftclient
+    pyyaml
+    requests
+    six
+  ];
+
+  checkInputs = [
+    stestr
+    testscenarios
+    requests-mock
+  ];
+
+  checkPhase = ''
+    stestr run -e <(echo "heatclient.tests.unit.test_common_http.HttpClientTest.test_get_system_ca_file")
```

Comment:
Please put the disabled test in its own line like I did to make diffs nicer in the future.